src/entrypoints/accounts-actions/accountOwner.ts

```diff
@@ -0,0 +1,36 @@
+import { HexString } from "ethers/lib.commonjs/utils/data";
+import deployments from "../../deployments";
+
+const SENTINEL_ADDRESS = "0x0000000000000000000000000000000000000001";
+const NUMBER_OF_SIGNERS_TO_FETCH = 10;
+
+const delayMod = deployments.delayModMastercopy;
+
+export function addAccountOwner(account: string) {
+  return delayMod.iface.encodeFunctionData("enableModule", [account]);
+}
+
+type EthCallCallback = (
+  encodedFunctionCall: `0x${string}`
+) => Promise<{ data: HexString | undefined }>;
+
+export async function getAccountOwners(
+  doEthCall: EthCallCallback,
+  numberOfSignersToFetch = NUMBER_OF_SIGNERS_TO_FETCH
+) {
+  const encodedFunctionCall = delayMod.iface.encodeFunctionData(
+    "getModulesPaginated",
+    [SENTINEL_ADDRESS, numberOfSignersToFetch]
+  ) as `0x${string}`;
+
+  const ethResult = await doEthCall(encodedFunctionCall);
```

Comment:
I would actually do these functions as invoker friendly as possible - I would just pass in doEthCall and then return an array - however long it is - with the signers. Internally I would figure how many signers there are, and etc.

It doesn't matter, an eth_call is free, just loop through chunks of fetching 100 or something like that. And assemble the full result